public/app/features/explore/utils/decorators.ts

```diff
@@ -20,6 +20,8 @@ import { ExplorePanelData } from '../../../types';
 import { CorrelationData } from '../../correlations/useCorrelations';
 import { attachCorrelationsToDataFrames } from '../../correlations/utils';
 import { sortLogsResult } from '../../logs/utils';
+import { hasPanelPlugin } from '../../plugins/importPanelPlugin';
+import { preProcessPanelData } from '../../query/state/runRequest';
```

Comment:
This is the issue:

the function `preProcessPanelDate` has moved to `@grafana/data`